core/store/src/migrations.rs

```diff
@@ -204,3 +204,25 @@ pub fn migrate_9_to_10(path: &String, is_archival: bool) {
     }
     set_store_version(&store, 10);
 }
+
+pub fn migrate_10_to_11(path: &String) {
+    let store = create_store(path);
+    let batch_size_limit = 100_000_000;
+    let mut store_update = store.store_update();
+    let mut batch_size = 0;
+    for (key, value) in store.iter_without_rc_logic(ColChunks) {
+        batch_size += key.len() + value.len() + 8;
+        let chunk: ShardChunk =
+            BorshDeserialize::try_from_slice(&value).expect("Borsh should not fail");
+        for receipt in chunk.receipts {
+            let bytes = receipt.try_to_vec().expect("Borsh should not fail");
+            store_update.update_refcount(ColReceipts, receipt.receipt_id.as_ref(), &bytes, 1);
+        }
```

Comment:
This means that we are going to store duplicates of all the receipts (one copy is inside the chunk, and another one is in a separate column). This is an increase in storage consumption, IO load, and CPU load during transaction processing. All of this is not required for regular NEAR node operation. According to our [JSON RPC policies](https://github.com/nearprotocol/nearcore/blob/master/chain/jsonrpc/README.md#policies-for-api-changes):

> We only add the APIs to the data that is already available in nearcore storage.

What is wrong with the Indexer approach? It was designed exactly to bridge the gap for such usecases, where we don't want to bloat nearcore.

/cc @nearmax @khorolets